BaseApi/terraform/staging/main.tf

```diff
@@ -60,3 +60,38 @@ module "development" {
   task_definition_secrets      = {}
   task_definition_secret_count = number # This number needs to reflect the number of environment variables provided
 }
+
+# THE SECTION BELOW IS FOR USE WITH DYNAMODB
```

Comment:
All these sections commented - should we keep them or uncomment them? (sorry, new to this version of organizing DynamoDB)

Reply:
New to me too.
I asked @LBHMGeorgieva about that. She advised me that it would be better to have a single template project that could support both Postgres and DynamoDb, but that the DynamoDb stuff would initially be commented out. A dev creating a new Api from the template could then use one and delete the other as needed.